ext/pdo/tests/pdo_test.inc

```diff
@@ -82,5 +69,27 @@ class PDOTest {
 
         return $config;
     }
+
+    public static function dropTableIfExists(PDO $db, string $tableName): void {
+        if (getenv('PDOTEST_DSN') === "sqlite::memory:") {
+            return;
+        }
+
+        match ($db->getAttribute(PDO::ATTR_DRIVER_NAME)) {
+            "oci" => $db->exec(<<<SQL
+                 BEGIN
+                     EXECUTE IMMEDIATE 'DROP TABLE $tableName';
+                 EXCEPTION
+                     WHEN OTHERS THEN
+                         IF SQLCODE != -942 THEN
+                             RAISE;
+                         END IF;
+                    END;
+                SQL
+            ),
+            "firebird" => @$db->exec("DROP TABLE $tableName"),
+            default => $db->exec("DROP TABLE IF EXISTS $tableName"),
```

Comment:
Dumb solution, how about using ``RECREATE`` then ``DROP`` for Firebird? This would work without using the ``@`` operator.

See: https://stackoverflow.com/a/49963337

Reply:
Very interesting workaround... ^^ I'm not sure if removing the `@` is worth the hassle 🤔